pact/ledger.pact

```diff
@@ -69,7 +69,40 @@
     @event true
   )
 
-  (defcap TOKEN:bool (id:string)
+  (defcap TOKEN:bool (id:string precision:integer supply:decimal policy:module{kip.token-policy-v1})
```

Comment:
`supply` doesn't seem necessary here, since it's always 0.0. What about we emit `TOKEN` instead of `SUPPLY` in 
https://github.com/kadena-io/marmalade/blob/1f6a52ea5fae517ca6f290580cbf7b9b6204e823/pact/ledger.pact#L418 
and remove SUPPLY?

Reply:
Mostly future-proofing for the frontend, we may allow a newly created token to have a non-zero supply later on and it's nice to have a fully populated (to the default) state. I'd keep supply, as it's much more common. `TOKEN` is basically create account for tokens